src/renderer/ScreenController.tsx

```diff
@@ -61,19 +58,13 @@ export default class ScreenController extends Component {
     this.userFeedback = this.userFeedback.bind(this)
     this.userFeedbackClick = this.userFeedbackClick.bind(this)
     this.openContextMenu = this.openContextMenu.bind(this)
-    this.openDialog = this.openDialog.bind(this)
     this.changeScreen = this.changeScreen.bind(this)
-    this.closeDialog = this.closeDialog.bind(this)
     this.onShowAbout = this.showAbout.bind(this)
     this.onShowKeybindings = this.showKeyBindings.bind(this)
     this.onShowSettings = this.showSettings.bind(this)
-    this.dialogController = createRef()
     this.selectAccount = this.selectAccount.bind(this)
 
-    window.__openDialog = this.openDialog.bind(this)
     window.__userFeedback = this.userFeedback.bind(this)
-    window.__closeDialog = this.closeDialog.bind(this)
-    window.__hasOpenDialogs = this.hasOpenDialogs.bind(this)
```

Comment:
Putting the dialog controls on the global window allowed for some neat developer hacks to access dialogues wherever one is but it also made it hard to grasp how things are tied together and where they came from